### DISCUSSION TITLE
TreeData does not produce a Bool_t in the output tree

### DISCUSSION THREAD

**Discussion by naidoo88** [dglazier/chanser#30] [User problems] [answered]
I'm hitting an issue where I have a missing variable in my output tree.  
It is included in my TreeData<FS>.h file with a default value:

`Bool_t flag_cut_3sigPi0IM = kFALSE;`

I have tried cleaning aux files and rerunning but no joy.

Cheers,
Paul

**Comment by dglazier** [comment, marked as answer]
This is true, TreeData does not take bools (I read somewhere once that they make a mess of the cache due to being 1bit fitting into 8bit space, probably it is not a big issue though) You can just make your variable a Short_t or Int_t type and be able to the same thing as if it was a bool.
If it is important to have bools these can be added...
